python/CMakeLists.txt

```
@@ -5,6 +5,11 @@ else()
 endif()
 cmake_policy(SET CMP0079 NEW)
 
+if(NOT DEFINED LIBINT_VERSION)
+    # for indep bld of project(libint2-python), version not available from project(Libint2)
+    set(LIBINT_VERISON "2.8.1")  # for version for setup.py from libtool+cmake
```

```suggestion
    set(LIBINT_VERSION "2.8.1")  # for version for setup.py from libtool+cmake
```
you'd think I'd have learned from the other branch ...